tools/merger/data/ci/ui/job.yml.template

```diff
@@ -1,6 +1,7 @@
   {{{job_name}}}:
     environment:
       GROUPAROO_TELEMETRY_ENABLED: "false"
+      NODE_ENV: development
```

Comment:
That's interesting! Why can't NODE_ENV be test?

Reply:
Their [docs suggested this](181234996-playwright-migration) so it would be able to pick up and install the right dependencies.

However, that was when using the Playwright Docker Image, which we aren't doing either.  I'll see if we can leave it alone.